src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.Standalone.csproj

```diff
@@ -13,10 +13,13 @@
     <OutputPath>bin\$(Configuration)\Console\build</OutputPath>
     <PublishDir>bin\$(Configuration)\Console\publish\$(RuntimeIdentifier)</PublishDir>
     <PublishTrimmed>true</PublishTrimmed>
-    <PublishSingleFile>true</PublishSingleFile>
     <AssemblyName>dd-trace</AssemblyName>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)' == ''">
+    <PublishSingleFile>true</PublishSingleFile>
```

Comment:
Does this prevent publishing? I thought this was just a build setting related to the Publish step?

Reply:
This prevent the error from VS when we are trying to build the project. 

What I'm doing is following instructions 😁:
```
C:\Program Files\dotnet\sdk\5.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(142,5): error NETSDK1097: It is not supported to publish an application to a single-file without specifying a RuntimeIdentifier. You must either specify a RuntimeIdentifier or set PublishSingleFile to false.
```

Reply:
Makes sense, thanks!